templates/accounts/resend_activation_mail.html

```diff
@@ -0,0 +1,29 @@
+{% extends 'layout.html' %}
+
+{% load static %}
+
+{% block title %} Jagrati | Resend activation mail {% endblock %}
+
+
+{% block content %}
+
+<div class="container pt-3" style="margin-top: 100px;">
+    <div class="text-center mt-4 p-0 p-sm-3" style="margin: auto; max-width: 500px;">
+        <h1 class="mt-2 font-weight-bolder" style="margin-bottom: 30px;">Resend Activation mail</h1>
```

Comment:
Instead of `style` and `mt-2` here, just put a `mb-4` class.

Start "activation" with a lower-case letter.